docs/content/preview/deploy/multi-dc/async-replication/async-transactional-failover.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Unplanned failover with transactional xCluster replication
+headerTitle: Unplanned failover
+linkTitle: Failover
+description: Unplanned failover using transactional (active-standby) replication between universes
+headContent: Failover of application traffic to the standby universe
+menu:
+  preview:
+    parent: async-replication-transactional
+    identifier: async-transactional-failover
+    weight: 30
+type: docs
+---
+
+Unplanned failover is the process of switching application traffic to the Standby universe in case the Primary universe becomes unavailable. One of the common reasons for such a scenario is an outage of the Primary universe region.
+
+## Performing failover
+
+Assuming universe A is the Primary and universe B is the Standby, use the following procedure to perform an unplanned failover and resume applications on B.
+
+### Pause replication and get the safe time
+
+If the Primary is Terminated for some reason, do the following:
+
+1. Stop the application traffic to ensure no more updates are attempted.
+
+1. Pause replication on the Standby (B). This step is required to avoid unexpected updates arriving through replication, which can happen if the Primary comes back up before the failover process is completed.
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses <standby_master_addresses> \
+        -certs_dir_name <cert_dir> \
+        set_universe_replication_enabled <replication_name> 0
+    ```
+
+    Expect output similar to the following:
+
+    ```output
+    Replication disabled successfully
+    ```
+
+1. Get the latest consistent time on Standby (B). The `get_xcluster_safe_time` command provides an approximate value for the data loss expected to occur when the unplanned failover process finishes, and the consistent timestamp on the Standby universe.
+
+    ```sh
+    yb-admin \
```

Comment:
Begin the command with `./bin/yb-admin` to maintain consistency.

Reply:
I see it at other places too. Replace all occurences of yb-admin to ./bin/yb-admin